src/setup.rs

```diff
@@ -135,7 +134,7 @@ pub(crate) fn initial_state(
         player_config,
         backend,
         initial_volume: config.initial_volume,
-        has_volume_ctrl,
+        disable_volume: false,
```

Comment:
I don't think, this is what we want. The previous flag will be ignored this way.